source/c_dataexchange.rst

```diff
@@ -0,0 +1,406 @@
+    <timestamp>,<timeseries_supplier_code or uuid>,<value>[\n]
+    <timestamp>,<timeseries_supplier_code or uuid>,<value>[\n]      
+    …
+
+Where:
+
+    * **timestamp:** time in UTC in ISO 8601 format. For example 2012-10-26T09:22:35Z. Supplying timestamps in different timezone is only allowed when the UTC offset is added to the timestamp according to ISO 8601. For example: 2012-10-26T07:22:35+02.
+    * **timeseries_supplier_code or UUID:** supplier_code attribute of timeseries as registered by administrator/supplier or the UUID of the timeseries object.
+    * **value:** numerical or textual value.
+    * **[\\n]:** newline character.
+
+CSV requirements:
+
+    * CSV file size may not exceed 100 MB. For one timeseries with a measuring frequency of 1 second that would be around 1 month of data.
+    * Every supplied file should contain new measurements. It is not allowed to add measurements to a previously supplied file.
+    * Use the empirical CSV format where the field separator is a comma (,) and the decimal separator a period (.).
+
+PiXML
++++++++++
+
+Pi-XML is a file exchange format used by Delft-FEWS. A description can be found `here <https://publicwiki.deltares.nl/display/FEWSDOC/Delft-Fews+Published+Interface+timeseries+Format+%28PI%29+Import>`_.
+
+Images
+++++++++++++
+
+The following formats are supported for image timeseries:
+
+    * PNG
+    * JPEG
+
+Image filenames should have the following format:
+
+.. code-block:: none
+
+    <timeseries_supplier_code>_<timestamp>.png
+    <timeseries_supplier_code>_<timestamp>.jpg
+
+Where:
+
+    * **timestamp:** time in UTC in ISO 8601 format. Colons (:) should be excluded as they are not allowed in filenames. An example timestamp is: 2012-10-26T092235Z. Supplying timestamps in different timezone is only allowed when the UTC offset is added to the timestamp according to ISO 8601. For example: 2012-10-26T072235+02.
+    * **timeseries_supplier_code:** supplier_code attribute of timeseries as registered by administrator/supplier.
+
+Video and multimedia
+++++++++++++++++++++++
+
+The following multimedia file extensions are supported for multimedia timeseries:
+
+    * PDF
+    * AVI
+    * WMV
+
+Multimedia filenames should have the following format:
+
+.. code-block:: none
+
+    <timeseries_supplier_code>_<timestamp>.pdf
+    <timeseries_supplier_code>_<timestamp>.avi
+    <timeseries_supplier_code>_<timestamp>.wmv
+
+Where:
+
+* **timestamp:** time in UTC in ISO 8601 format or as a unix timestamp in ms since epoch. Colons (:) should be excluded as they are not allowed in filenames. An example timestamp is: 2012-10-26T092235Z. Supplying timestamps in different timezone is only allowed when the UTC offset is added to the timestamp according to ISO 8601. For example: 2012-10-26T072235+02.
+* **timeseries_supplier_code:** supplier_code attribute of timeseries as registered by administrator/supplier.
+
+LMW data
++++++++++++
+
+Every 10 minutes a file is downloaded from LMW.
```

Comment:
Afko? Zeker buitenlanders zullen geen flauw idee hebben wat een Local Management Waterboard nou weer is.